README.md

```diff
@@ -1,2 +1,81 @@
 # eth-test-raw
-Tool to test Ethernet interfaces with raw ethernet packets
+Simple test tool for ethernet interfaces.
+
+Tests a ethernet NIC against a NIC on a peer machine using raw ethernet packets.
+Requires the corresponding server running on the peer machine.
+
+The purpose of this tool is to test the stability of the NIC hardware, for example to use it for environmental qualification tests. It is not optimized for throughput.
+
+## Platform compatibility
+Tested only under Linux (Rasperian OS and Yocto arm64). May work on Windows.
+
+## How to Use
+
+Connect device with the NIC to be tested to a peer machine via Ethernet.
+You may have L2 switches in between, but no routers.
+
+Deploy this repository, or at least the folders `client`, `server` and common to both machines.
+Let's say, you have it deployed under ~/eth-raw-test.
+
+### On the Server
+You need to know the interface name of the peer machine's NIC, e.g. `eth0`.
+
+Find out the MAC address of the peer machine's NIC:
+```bash
+ip link show eth0
+```
+Record the value for `link/ether`, e.g. `dc:a6:32:51:0e:e7`
+
+Start server on peer machine with root priviledges. Specify the name of the peer machine's NIC:
+```bash
+python3 server/eth-test-raw-server.py eth0
```

Comment:
changed folder structure, call should be "python3 eth-test-raw-server.py eth0"

Reply:
ok